src/main/java/seedu/address/model/location/Location.java

```diff
@@ -131,7 +131,6 @@ public String toString() {
     public boolean equals(Object other) {
         return other == this
                 || (other instanceof Location
-                && name.equals(((Location) other).name)
                 && Math.abs(lat - ((Location) other).lat) <= ALLOWABLE_ERROR
                 && Math.abs(lon - ((Location) other).lon) <= ALLOWABLE_ERROR);
```

Comment:
Repetitive explicit type cast.  You can typecast once before the return statement.

Reply:
Alright.